test/src/org/labkey/test/tests/response/TokenValidationTest.java

```diff
@@ -227,20 +231,22 @@ public void testMyStudiesCoordinatorRole()
         // Test for Administrator
         assertTrue(batchesWebPart.isNewBatchPresent());
         assertTrue(batchesWebPart.isNewBatchEnabled());
-        setupPage.validateSubmitButtonDisabled();  // Submit button should be present
+        ForwardingTab forwardingTab = ForwardingTab.beginAt(this);
+        forwardingTab.validateSaveButtonDisabled();  // Save button should be present
+        SetupPage.beginAt(this, PROJECT_NAME01);
 
         // Test for Reader
         impersonateRole("Reader");
         assertTrue(batchesWebPart.isNewBatchPresent());
         assertFalse(batchesWebPart.isNewBatchEnabled());
-        assertFalse(setupPage.isSubmitButtonVisible());  // Submit button should NOT be present
+        ForwardingTab.beginAt(this);
+        assertTextPresent("User does not have permission to perform this operation.");  // Not authorized
         stopImpersonating(false);
 
         // Test for MyStudies Coordinator
         impersonateRoles("Reader", "MyStudies Coordinator");
         assertTrue(batchesWebPart.isNewBatchPresent());
         assertTrue(batchesWebPart.isNewBatchEnabled());  // Should be able to create a new batch
-        assertFalse(setupPage.isSubmitButtonVisible());  // Submit button should NOT be present
```

Comment:
UI is now located in Folder Management, which a MyStudies Coordinator will not have access to